_resources/port1.0/group/python-1.0.tcl

```diff
@@ -447,3 +495,20 @@ pre-livecheck {
         livecheck.type  none
     }
 }
+
+
+pre-test {
+    # disable test phase for py-<portname> stub ports
+    if {[string match py-* [option name]] && ${name} eq ${subport}} {
+          test.run      no
+    }
+
+    # if using PEP517, unzip the .whl file to ${worksrcpath}/build/lib
+    if {${python.pep517}} {
+        file mkdir ${worksrcpath}/build/lib
+        system -W ${workpath} "unzip -o *.whl -d [shellescape ${worksrcpath}/build/lib]"
+    }
+
+    # set PYTHONPATH
+    test.env-append PYTHONPATH=[glob -nocomplain ${worksrcpath}/build/lib*]
```

Comment:
Technically, `glob` returns a list. I don't know if there will ever be more than one match in practice, but to be completely correct I guess this should be `[join [glob …] :]`?

Reply:
I *think* there should never be more than one `lib` directory, but I made the change as it doesn't hurt either